addOns/zest/src/main/java/org/zaproxy/zap/extension/zest/ZestStatementFromJson.java

```diff
@@ -52,81 +55,59 @@ public class ZestStatementFromJson {
     private static final String ZEST_CLIENT_LAUNCH = "ZestClientLaunch";
     private static final String ZEST_CLIENT_ELEMENT_CLICK = "ZestClientElementClick";
     private static final String ZEST_CLIENT_ELEMENT_SEND_KEYS = "ZestClientElementSendKeys";
+    private static final String ZEST_CLIENT_ELEMENT_SUBMIT = "ZestClientElementSubmit";
     private static final String ZEST_CLIENT_ELEMENT_CLEAR = "ZestClientElementClear";
     private static final String ZEST_CLIENT_WINDOW_CLOSE = "ZestClientWindowClose";
     private static final String ZEST_CLIENT_SWITCH_TO_FRAME = "ZestClientSwitchToFrame";
     private static final String ZEST_CLIENT_WINDOW_RESIZE = "ZestClientWindowResize";
 
     public static ZestStatement createZestStatementFromJson(JSONObject json) throws Exception {
-        ZestStatement stmt = null;
-        if (json.containsKey(ELEMENT_TYPE) && json.containsKey(INDEX)) {
-            int index = json.getInt(INDEX);
+        if (json.containsKey(ELEMENT_TYPE)) {
             String elementType = json.getString(ELEMENT_TYPE);
-            switch (elementType) {
-                case ZEST_CLIENT_LAUNCH:
```

Comment:
Can these changes be done in another commit? They make it more difficult to see the actual fixes.

Reply:
There was me thinking I was being helpful 😛